app/src/components/governance/PageProposal.vue

```diff
@@ -227,7 +231,12 @@ export default {
       /* istanbul ignore next */
       skip() {
         // only Tendermint networks have this network-wide "governance parameters" logic
-        return !this.found || this.currentNetwork.network_type !== `cosmos`
+        return (
+          !this.found ||
+          this.currentNetwork.network_type !== `cosmos` ||
+          this.currentNetwork.id === `emoney-mainnet` ||
```

Comment:
why do parameters not work in emoney?

Reply:
no governance

Reply:
why don't you use the general purpose feature flag for that?

Reply:
how does a user even get to this page for money?

Reply:
I also think this part is not necessary